processor/src/main/kotlin/io/github/doljae/kotlinlogging/extensions/LoggerProcessor.kt

```diff
@@ -12,6 +12,55 @@ class LoggerProcessor(
     private val codeGenerator: CodeGenerator,
     private val logger: KSPLogger,
 ) : SymbolProcessor {
+    companion object {
```

Comment:
The location of the companion object is probably best at the bottom of the class, since its inner workings are `private`. It's probably a good idea to put your class's public API at the top.

I realized I didn't specify a separate code style convention, so I'll add that separately 🙂